packages/ERTP/src/paymentLedger.js

```diff
@@ -20,14 +20,26 @@ import '@agoric/store/exported';
  * @param {Brand} brand
  * @param {AssetKind} assetKind
  * @param {DisplayInfo} displayInfo
+ * @param {ShutdownWithFailure=} optShutdownWithFailure
  * @returns {{ issuer: Issuer, mint: Mint }}
  */
 export const makePaymentLedger = (
   allegedName,
   brand,
   assetKind,
   displayInfo,
+  optShutdownWithFailure = undefined,
 ) => {
+  /** @type {ShutdownWithFailure} */
+  const shutdownLedgerWithFailure = reason => {
```

Comment:
It seems clunky to have to define this here, but it's non-blocking.

Reply:
Ok. I'll merge as is but am interested in hearing suggestions